jscomp/core/lam_pass_remove_alias.ml

```diff
@@ -256,6 +257,18 @@ let simplify_alias (meta : Lam_stats.t) (lam : Lam.t) : Lam.t =
                             (Lam_beta_reduce.propagate_beta_reduce_with_map meta
                                param_map params body ap_args))
                 | _ -> normal ()
+                in
+                let result = (match result with
+                | Lprim {primitive; args; loc} -> (match primitive with
```

Comment:
formatting looks a little weird

Reply:
hmm for some reason autoformat is broken in my setup, I'll try to fix it.

Reply:
There's no auto format. Long story.
For now manual format